java/java.sourceui/src/org/netbeans/api/java/source/ui/ElementJavadoc.java

```diff
@@ -1280,6 +1281,19 @@ private StringBuilder inlineTags(List<? extends DocTree> tags, TreePath docPath,
                 case TEXT:
                     TextTree ttag = (TextTree)tag;
                     sb.append(ttag.getBody());
+					break;
+				default:
+                    if (tag.getKind().toString().equals("SNIPPET")) {
+                        sb.append("<pre>"); //NOI18N
+                        sb.append("<code>"); //NOI18N
+                        List<DocTree> attributes = TreeShims.getSnippetDocTreeAttributes(tag);
```

Comment:
"attributes" seems unused

Reply:
This is being modified in further commits (0a06b63907ad297e9b53b16e7a019f5ffcf0ade1)